egs/malach/s5/README.txt

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019  IBM Corp. (Author: Michael Picheny) Adapted AMI recipe to MALACH corpus
+
+This s5 recipe for MALACH data is a modified version of the s5b
+recipe for AMI. 
+
+You need to download the malach data to get started. For information about the MALACH database see : 
+USC-SFI MALACH Interviews and Transcripts English - Speech Recognition Edition
+https://catalog.ldc. upenn.edu/LDC2019S11
```

Comment:
Th espace in this URL makes it not work.  You might want to consider renaming this to malach_english and clarifying in this file that it's English.  That will make it more discoverable for people who are looking for English data.  Up to you though.